1-exercises/A-this/exercise-3.js

```diff
@@ -8,8 +8,12 @@
 
 let bankAccount = {
   currentAmount: 100,
-  deposit: function (amount) {},
-  withdraw: function (amount) {},
+  deposit: function (amount) {
+    this.currentAmount += amount;
+  },
+  withdraw: function (amount) {
+    this.currentAmount -= amount;
+  },
```

Comment:
Good use of plus equal and minus equal to increment and decrement

Reply:
Thank you